vsphere/datadog_checks/vsphere/config.py

```diff
@@ -141,6 +141,10 @@ def __init__(self, instance, init_config, log):
         )
         self.include_datastore_cluster_folder_tag = instance.get("include_datastore_cluster_folder_tag", True)
         self.custom_tags = instance.get('tags', [])
+        self.hostname_to_upper = instance.get('hostname_to_upper', False)
+        self.hostname_to_lower = instance.get('hostname_to_lower', False)
```

Comment:
```suggestion
        self.hostname_to_upper = is_affirmative(instance.get('hostname_to_upper', False))
        self.hostname_to_lower = is_affirmative(instance.get('hostname_to_lower', False))
```
nit- to take into account all possible affirmative values

Reply:
Changed the config option to `hostname_transform`, where is_affirmative() wouldn't apply.